### DISCUSSION TITLE
Scroll Text buffer without Keyboard enabled - edit: when using editor vi

### DISCUSSION THREAD

**Discussion by cs-hv** [isontheline/pro.webssh.net#1255] [Ideas]
It would be great if the text could be scrolled when displayed in full screen (i.e. keyboard is hidden). A floating navigation element might be a possible solution to scroll up/down and left/right?

**Comment by isontheline** [comment]
Hello @cs-hv 👋

It’s already the case. 
Could you please share video if not?

**Comment by cs-hv** [comment]
Hi @isontheline 
thank your for your prompt reply!
I have to admit I forgot one major thing: The issue occurs when using an editor like e.g. "vi" or "vim".
It would be perfect to provide a cursor control without having to display the keyboard which covers a huge part of the screen.
I edited the title accordingly.

**Comment by isontheline** [comment]
On iOS I would like to use : https://github.com/isontheline/pro.webssh.net/issues/1007#issue-1996251851

Scrolling with fingers is harder because in vi you’re using the alternate buffer

**Comment by cs-hv** [comment]
As I understand #1007, this would again require the keyboard to be visible (at least the space bar), right?

**Reply by isontheline** [reply]
😅 Sorry, you’re right

Try the old pad : « Virtual Arrows Key Type » inside the WebSSH / SSH settings

A arrow pad will appear

**Reply by cs-hv** [reply]
Perfect! That's what I was looking for.
